project/plugins.sbt

```diff
@@ -1,14 +1,23 @@
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.5.0")
+resolvers += Resolver.url("bintray-sbt-plugin-releases", url("http://dl.bintray.com/banno/oss"))(
+  Resolver.ivyStylePatterns
+)
 
-addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "2.3.0")
+resolvers += "sonatype-releases" at "https://oss.sonatype.org/content/repositories/releases/"
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.11.2")
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.3")
```

Comment:
This line needs to be removed. Otherwise scalariform still runs after the tests.

Reply:
good catch. I added  a commit to remove it.